serverlessworkflow/modules/ROOT/pages/eventing/working-with-callbacks.adoc

```diff
@@ -93,13 +93,13 @@ An link:{kogito_sw_examples_url}/serverless-workflow-callback-quarkus/src/main/j
     }
 ----
 
-After that, the Serverless Workflow application consumes the event published by the listener and sets the result field. The consumed CloudEvent contains an attribute named `kogitoprocrefid`, which holds the process instance ID of the workflow.
+After that, the workflow application consumes the event published by the listener and sets the result field. The consumed CloudEvent contains an attribute named `kogitoprocrefid`, which holds the process instance ID of the workflow.
 
 The `kogitoprocrefid` attribute is crucial because when the correlation is not used, then this attribute is the only way for the Callback state to identify that the related CloudEvent needs to be used to resume the workflow.
 
-Note that each Serverless Workflow is identified by a unique instance ID, which is automatically included in any published CloudEvent, as `kogitoprocinstance` CloudEvent attribute.
+Note that each workflow is identified by a unique instance ID, which is automatically included in any published CloudEvent, as `kogitoprocinstance` CloudEvent attribute.
 
-The following example shows that the event listener takes the process instance ID of a Serverless Workflow from a CloudEvent attribute named `kogitoprocinstanceid`, which is associated with the CloudEvent that is consumed. 
+The following example shows that the event listener takes the process instance ID of a workflow from a CloudEvent attribute named `kogitoprocinstanceid`, which is associated with the CloudEvent that is consumed. 
```

Comment:
Shouldn't we use the workflow instance ID instead of the process instance ID? There are other places at least in this file where it is mentioned.

Reply:
I agree. We should replace process instance ID with workflow instance ID.

Reply:
Done, changed process instance ID to workflow instance ID